dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguageSubclass.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ *  *    Copyright 2020 OICR
+ *  *
+ *  *    Licensed under the Apache License, Version 2.0 (the "License");
+ *  *    you may not use this file except in compliance with the License.
+ *  *    You may obtain a copy of the License at
+ *  *
+ *  *        http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  *    Unless required by applicable law or agreed to in writing, software
+ *  *    distributed under the License is distributed on an "AS IS" BASIS,
+ *  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  *    See the License for the specific language governing permissions and
+ *  *    limitations under the License.
+ *
+ */
+
+package io.dockstore.common;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum DescriptorLanguageSubclass {
+    DOCKER_COMPOSE("Docker Compose", "docker-compose"),
+    HELM("Helm", "helm"),
+    SWARM("Swarm", "swarm"),
+    KUBERNETES("Kubernetes", "kubernetes");
+
+    private final String friendlyName;
+
+    private final String shortName;
+
+    DescriptorLanguageSubclass(final String friendlyName, final String shortName) {
+        this.friendlyName = friendlyName;
```

Comment:
friendly name and short names aren't too different here, would recommend just picking one